FileInfoLoader.cc

```diff
@@ -443,11 +446,13 @@ bool FileInfoLoader::fillHdf5ExtFileInfo(FileInfoExtended* extendedInfo, string&
         crDegStr = fmt::format("[{}, {}]", cr1, cr2);
         if (!(cdelt1 == 0.0 && cdelt2 == 0.0))
             axisInc = fmt::format("{}, {}", unitConversion(cdelt1, cunit1), unitConversion(cdelt2, cunit2));
+        if (!(bmaj == 0.0 && bmin == 0.0 && bpa == 0.0))
```

Comment:
if either bmaj or bmin are zero, we don't really have beam info, surely?

Reply:
none of them should be zero, otherwise a beam cannot be defined.

Reply:
bpa can be zero, right? 

the current check only guards against all three of them being zero.

Reply:
Oops, right. None of bmaj and bmin should be zero. bpa can be zero.